src/material-experimental/mdc-input/testing/input-harness.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {MatInputHarness as BaseInputHarness} from '@angular/material/input/testing';
+
+/** Harness for interacting with a MDC-based Material inputs in tests. */
+export class MatInputHarness extends BaseInputHarness {}
```

Comment:
Need to add the static properties (`hostSelector`, `with`) if you do it this way. Another idea though: what if you just `export {MatInputHarness} from '@angular/material/input/testing'` since there's no difference

Reply:
Let's just re-export it for now. I had it that way before since I had the different `hostSelector`. 

I think we wouldn't necessarily need to re-create `hostSelector` since it will be inherited, but the `with` method would need to be copied so that it doesn't reference the inherited class.

Reply:
Done.